src/services/timers.py

```diff
@@ -0,0 +1,72 @@
+
+import datetime
+import time
+import traceback
+from typing import Tuple
+from infra import logger, utils 
+from infra.context import Context
+utils.load_env()
```

Comment:
We already load the environment at time 0 via src/run.py.
Why is it needed here?

Reply:
because it's a load of a different thread

Reply:
Shouldn't matter, env variables are per processes.

Just make sure the first thing src/run.py does is load the env, and this isn't needed.